src/pages/home/report/ReportActionItem.js

```diff
@@ -1,23 +1,51 @@
 import React from 'react';
+import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import ReportActionItemSingle from './ReportActionItemSingle';
 import ReportActionPropTypes from './ReportActionPropTypes';
 import ReportActionItemGrouped from './ReportActionItemGrouped';
+import getReportActionItemContainerStyles from '../../../styles/getReportActionItemContainerStyles';
+import styles from '../../../styles/styles';
+import ReportActionContextMenu from './ReportActionContextMenu';
+import Hoverable from '../../../components/Hoverable';
 
 const propTypes = {
+    // The ID of the report this action is on.
+    reportID: PropTypes.number.isRequired,
+
     // All the data of the action item
     action: PropTypes.shape(ReportActionPropTypes).isRequired,
 
     // Should the comment have the appearance of being grouped with the previous comment?
     displayAsGroup: PropTypes.bool.isRequired,
+
+    // Sets this ReportAction as the active (hovered) action on the report.
+    setIsActive: PropTypes.func.isRequired,
 };
 
 const ReportActionItem = props => (
-    <>
-        {!props.displayAsGroup
-            ? <ReportActionItemSingle action={props.action} />
-            : <ReportActionItemGrouped action={props.action} />}
-    </>
+    <Hoverable
+        onHoverIn={() => props.setIsActive(true)}
+        onHoverOut={() => props.setIsActive(false)}
+    >
+        {hovered => (
+            <View>
+                <View style={getReportActionItemContainerStyles(hovered)}>
+                    {!props.displayAsGroup
+                        ? <ReportActionItemSingle action={props.action} />
+                        : <ReportActionItemGrouped action={props.action} />}
+                </View>
+                <View style={styles.miniReportActionContextMenuWrapperStyle}>
+                    <ReportActionContextMenu
+                        reportID={props.reportID}
+                        reportActionID={props.action.sequenceNumber}
+                        shouldShow={hovered}
```

Comment:
Thoughts on removing this prop altogether and doing the conditional render here instead of inside the component? Might clean things up a bit since you are just returning `null` when this is `false` anyway.